src/command_modules/azure-cli-role/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+(unreleased)
+^^^^^^^^^^^^^^^^^^
+* add commands help for "az ad app/sp/user"
```

Comment:
I don't think you need to add a release note for updating the help text, since there's no bug being fixed or behavior being changed.